### ISSUE TITLE
Does not work on Mac

### ISSUE THREAD

**Issue by MarcusSjolin** [roramirez/MagicMirror-Module-Template#3]
There's a problem running the create_module.sh script on Mac.

This seem to be due to version missmatch of SED. When running the same command in an Ubuntu Docker container it works fine.

```
$ bash -c "$(curl -sL https://raw.githubusercontent.com/roramirez/MagicMirror-Module-Template/master/create_module.sh)"
MagicMirror installation not found.
Please input its path now (or restart script in its directory): /path
Insert your module name: MyModule
Do you want create in /path/module/MyModule (y/N) y
Pickup a license
  1. MIT (Default)
  2. ISC
Choice: 1
Type a short description of what your module does (leave empty to insert a Todo)
: Todo
Cloning into '/var/folders/s9/tkvlqcy57w37x46r69d2x9b40000gn/T/tmp.jUQrYWPk'...
remote: Counting objects: 160, done.
remote: Total 160 (delta 0), reused 0 (delta 0), pack-reused 160
Receiving objects: 100% (160/160), 22.42 KiB | 0 bytes/s, done.
Resolving deltas: 100% (82/82), done.
Checking connectivity... done.
sed: 1: "/path/module/MyModule ...": invalid command code m
sed: 1: "/path/module/MyModule ...": invalid command code m
sed: 1: "/path/module/MyModule ...": invalid command code m
sed: 1: "/path/module/MyModule ...": invalid command code m
sed: 1: "/path/module/MyModule ...": invalid command code m
Initialized empty Git repository in /path/module/MyModule/.git/
```

Running this manually shows me there's an error in CHANGELOG.md but that makes no sense at all.

```
sed -i s/\{\{MODULE_NAME\}\}/MyModule/g module/MyModule/*.*
sed: 1: "CHANGELOG.md": invalid command code C
```

**Comment by roramirez** [comment]
What versión of sed  are you've in your Mac?

**Comment by MarcusSjolin** [comment]
I'm afraid that answer is a bit complex,

There's an issue that you cannot really update sed, instead you can install gsed (gnu sed) and alias it to sed. That's about as close as it gets, but that's not always working either, especially in shell scripts etc.

http://stackoverflow.com/questions/37639496/how-can-i-check-the-version-of-sed-in-os-x

```
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

**Comment by MarcusSjolin** [comment]
Tried a few alterations.

This seem to solve the problem for Mac users:
http://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x

`sed -i '' s/\{\{MODULE_NAME\}\}/$MODULE_NAME/g`

**Comment by roramirez** [comment]
Please send a Pull Request and I can review it.

**Comment by roramirez** [comment]
Hi @MarcusSjolin 

Did you have any update about this?

**Comment by roramirez** [comment]
Fixed  by #17
